src/client_side.cc

```diff
@@ -1524,11 +1524,10 @@ bool ConnStateData::serveDelayedError(Http::Stream *context)
             bool allowDomainMismatch = false;
             if (Config.ssl_client.cert_error) {
                 ACLFilledChecklist check(Config.ssl_client.cert_error, nullptr);
-                check.sslErrors = new Security::CertErrors(Security::CertError(SQUID_X509_V_ERR_DOMAIN_MISMATCH, srvCert));
+                const auto sslErrors = std::make_unique<Security::CertErrors>(Security::CertError(SQUID_X509_V_ERR_DOMAIN_MISMATCH, srvCert));
```

Comment:
This `std::make_unique<CertErrors>(CertError(...))` code is fairly basic but duplicated. I did not add a function to encapsulate it, primarily because we should be reference-counting these errors _instead_ (and implementing that requires serious out-of-scope changes). I did add a TODO about reference counting.

Reply:
Given that the new type of `check.sslErrors` performs cdata/reference counting, why the additional `std::make_unique`?

Reply:
I see. Thank you.